### ISSUE TITLE
Typescript support

### ISSUE THREAD

**Issue by saurabhnemade** [ckeditor/ckeditor4-react#82]
I was unable to find types for ckeditor4-react. However types for ckeditor4 are available.
Can we get types for ckeditor4-react?

**Comment by NathanCH** [comment]
If you would like to declare your own typing:

```
declare module 'ckeditor4-react' {
  import * as React from 'react';

  export interface CKEditorProps {
    data: string;
  }

  export default class CKEditor extends React.Component<CKEditorProps, any> {}
}
```

**Comment by MMMalik** [comment]
I think we can easily add declaration file to our repo, so that TS users don't have to provide that themselves. What do you think @f1ames?

**Comment by f1ames** [comment]
> I think we can easily add declaration file to our repo, so that TS users don't have to provide that themselves. What do you think @f1ames?

Makes sense, especially that it should be rather straightforward :+1: Added to planning pipeline.

**Comment by apoorvkk** [comment]
+1, I am very keen to see typings in this library.

**Comment by f1ames** [comment]
Hi all,

This is already covered in new RC version available [already on NPM](https://www.npmjs.com/package/ckeditor4-react). Feel free to check how it works - your feedback on this is more than welcome. You can find documentation on a new version [here](https://ckeditor.com/docs/ckeditor4/latest/guide/dev_react_v2.html).

**Comment by saurabhnemade** [comment]
Thanks @f1ames . Will check it. 👍

**Comment by f1ames** [comment]
Hi again,

The RC version previously mentioned in https://github.com/ckeditor/ckeditor4-react/issues/82#issuecomment-878347355 has been released as `stable` `2.0.0` version today so you can use it for production.

Fixed in `2.0.0` version (https://github.com/ckeditor/ckeditor4-react/commit/f03dde821f2d5b62d46c4a2e80345aae2d7f39f0 and https://www.npmjs.com/package/ckeditor4-react/v/2.0.0).
